content/monad-transformers.md

```diff
@@ -23,13 +23,96 @@ A `Reader r a` in which the resulting `a` is wrapped in any other monad, i.e. `r
 
 ### StateT
 
-A `State s a` in which the return value `a` and state `s` are wrapped in any other monad, i.e. `s -> m (a, s)`
+A `State s a` in which the return value and state, namely `(a, s)`, are wrapped in any other monad, i.e. `s -> m (a, s)`
 
-### EitherT
+### ExceptT
 
 An `Either e a` wrapped in any other monad, i.e. `m (Either e a)`
 
-* Simple examples of usage
+## Simple examples of usage
+
+[transformers](https://www.stackage.org/package/transformers) is a widely used package which provides transformer versions of various monads. It also provides two useful classes, `MonadTrans` and `MonadIO`. 
+
+`MonadTrans` makes it easy to embed one monad into another. All of the transformers defined in the `transformers` package are instances `MonadTrans`.  
```

Comment:
This phrasing is somewhat inaccurate, and possibly confusing. A very strict explanation is that "`MonadTrans` represents the ability to lift a monadic action from a base monad into a transformed monad," but that's a mouthful and pretty opaque for beginners. Perhaps "instances of `MonadTrans` are transformers which can be applied to other monads to create new monads," or something of that ilk?